packages/neutrino/bin/neutrino

```diff
@@ -55,57 +46,32 @@ const args = yargs
   .help()
   .argv;
 
-function run(command, presets) {
-  process.env.NODE_ENV = environments[command];
-
+function run(command, args) {
+  const environments = { build: 'production', start: 'development', test: 'test' };
+  const pkg = optional(join(process.cwd(), 'package.json')) || {};
+  const pkgPresets = pathOr([], ['neutrino', 'presets'], pkg);
+  const presets = [...new Set(pkgPresets.concat(args.presets))];
   const options = pathOr({}, ['neutrino', 'options'], pkg);
   const config = pathOr({}, ['neutrino', 'config'], pkg);
   const api = new Neutrino(Object.assign(options, { config }));
+  const inspect = pipe(sort, partialRight(stringify, [null, 2]), console.log, process.exit);
 
-  // Grab all presets and merge them into a single webpack-chain config instance
-  presets.forEach(preset => {
-    const paths = [
-      join(cwd, preset),
-      join(cwd, 'node_modules', preset),
-      preset
-    ];
-
-    for (let i = 0; i < paths.length; i += 1) {
-      try {
-        return api.use(require(paths[i]));
-      } catch (exception) {
-        if (/Cannot find module/.test(exception.message)) {
-          continue;
-        }
-
-        exception.message = `Neutrino was unable to load the module '${preset}'. ` +
-          `Ensure this module exports a function and is free from errors.\n${exception.message}`;
-        throw exception;
-      }
-    }
+  process.env.NODE_ENV = environments[command];
 
-    throw new Error(`Neutrino cannot find a module with the name or path '${preset}'. ` +
-      `Ensure this module can be found relative to the root of the project.`);
-  });
+  // Grab all presets and merge them into a single webpack-chain config instance
```

Comment:
I just had a thought. We are using two different nomenclatures for _items_ we are requiring: "presets" and "middleware". Presets are a carryover from pre-v5, and presets are technically middleware. Instead of naming things by _what_ we are loading, what if we named them by the _action_ we are going to take on them?

**use**

We already do this in the API. We _use_ presets. We _use_ middleware. The CLI interface should be synonymous with the API action.

So I'm proposing to change the CLI flag from `--presets` to `--use`. Would you make that change along with this? Then the package.json object would also change from `neutrino.presets : []` to `neutrino.use : []`.

That would inform the name of this methods better too:

```js
api
  .requireMiddleware(middleware)
  .forEach(middleware => api.use(middleware))
```

Or maybe `requireMiddleware` is too verbose, and the only thing we ever require **is** middleware, so it can be simply `api.require` or `api.import`. One thing I like about the connotation of `import` is that it presupposes `use`:

```js
middleware.forEach(middleware => api.import(middleware)) // Does a require AND a use
```

TLDR; Let's change the CLI to use `--use`, change `neutrino.presets` to `neutrino.use` for package.json, use the variable of `middleware` instead of `presets`, and create `Neutrino#import` which does a `require` and a `use`.